components/HeroHeader/HeroHeader.js

```diff
@@ -63,7 +65,9 @@ const HeroHeader = ({ authenticated }) => {
                     )}
                     <Wrap href="/bills">
                       <div className={styles.btncontainer}>
-                        <Button variant="outline-secondary">Browse</Button>
+                        <Button variant="outline-secondary">
+                          {capitalize(t("browse"))}
```

Comment:
This is a super nit but I'm kind of curious if [`capitalize`](https://lodash.com/docs/4.17.15#capitalize) is locale-aware. It doesn't look like it from the description... not sure if every language capitalizes the first letter or if there are some funky rules.

Reply:
intense question. I will put this in the epic also.